plugin/SearchBarItem.qml

```diff
@@ -162,11 +183,12 @@ BackgroundItem {
         EnterKey.onClicked: {
             if (text != "") {
                 _searchText = text
+                focus = false
                 root.requestSearch(text)
             } else {
+                focus = false
```

Comment:
I've added a comment. I don't like much having it in the two branches, but the call in the first branch must appear before requesting search, because loosing the focus will use matchCount property which may be changed by the request. But at the same time, it must be set after saving current text into _searchText, because loosing the focus will reset text (matchCount being zero on a starting search).